packages/nimble-components/src/button/tests/button-matrix.stories.ts

```diff
@@ -81,3 +81,11 @@ export const hiddenButton: StoryFn = createStory(
 export const textCustomized: StoryFn = createMatrixThemeStory(
     textCustomizationWrapper(html`<${buttonTag}>Button</${buttonTag}>`)
 );
+
+export const inlineAlignment: StoryFn = createStory(
```

Comment:
I think the test will make more sense to future visitors if there was a comment along the lines of "The baseline of the elements should be aligned when positioned inline" and then if we visualized the underline:

```ts
<span style="text-decoration: underline;">Text</span>
<nimble-button current-value="" appearance="outline"><span style="text-decoration: underline;">Button</span></nimble-button>
<span style="text-decoration: underline;">Text</span>
<nimble-button appearance="outline"><nimble-icon-key slot="start"></nimble-icon-key><span style="text-decoration: underline;">Button</span></nimble-button>
<div style="display:inline-block; width:50px; height:50px; border: 1px black solid;"></div>
```

![image](https://github.com/ni/nimble/assets/1588923/c5325aa2-2c02-43ff-9667-6bcebb468110)

Reply:
Although that doesn't really align that great. I wonder if it is better if the span text font between the buttons is the same as the buttonLabelFont